### PR TITLE
Resolve merge conflicts in (Pass the language code to iTunes search api to get localized release notes) PR

### PR DESCRIPTION
**Pull request by ibraheemalayan**

### CODE REVIEW

**Comment by ibraheemalayan** [comment]
resolve merge conflicts from PR #351

**Comment by larryaasen** [comment]
This PR seems similar to https://github.com/larryaasen/upgrader/pull/443.

File: lib/src/itunes_search_api.dart [hunk 1]
```diff
@@ -32,14 +32,16 @@ class ITunesSearchAPI {
...
+      {String? country = 'US',String? language = 'en', bool useCacheBuster =
+      true}) async {
     assert(bundleId.isNotEmpty);
     if (bundleId.isEmpty) {
       return null;
     }
 
     final url = lookupURLByBundleId(bundleId,
-        country: country ?? '', useCacheBuster: useCacheBuster)!;
+        country: country ??= '', language: language ?? '', useCacheBuster:
```
Comment (larryaasen) [inline lib/src/itunes_search_api.dart:43]: @ibraheemalayan There seems to be an extra character added to this existing code `country: country ??= ''`. The `=` was added. Was there a reason for thiat?

Comment (ibraheemalayan) [reply lib/src/itunes_search_api.dart:43]: Looks like it is a mistake but it doesn't affect the code.

Comment (larryaasen) [reply lib/src/itunes_search_api.dart:43]: I can merge this after that extra character is removed.

Comment (ibraheemalayan) [reply lib/src/itunes_search_api.dart]: I did the change

**Comment by ibraheemalayan** [comment]
@larryaasen 

Is the test that failed related ?

**Comment by larryaasen** [comment]
> @larryaasen
> 
> Is the test that failed related ?



> @larryaasen
> 
> Is the test that failed related ?



> @larryaasen
> 
> Is the test that failed related ?

@ibraheemalayan 
 It was not a test that failed, it was `flutter analyze` for Flutter 3.24.0

**Comment by larryaasen** [comment]
@ibraheemalayan Do you have Flutter 3.24.0+ on your local machine? If not, I can fix this issue.

**Comment by larryaasen** [comment]
> @ibraheemalayan Do you have Flutter 3.24.0+ on your local machine? If not, I can fix this issue.

I will fit it here. Thanks for the PR 👍

**Comment by larryaasen** [comment]
These updates were published on pub.dev as version 11.1.0: https://pub.dev/packages/upgrader/versions/11.1.0

**Pull request merged**